model/project.go

```diff
@@ -911,9 +912,10 @@ func GetTaskGroup(taskGroup string, tc *TaskConfig) (*TaskGroup, error) {
 	if taskGroup == "" {
 		// if there is no named task group, fall back to project definitions
 		return &TaskGroup{
-			SetupTask:    p.Pre,
-			TeardownTask: p.Post,
-			Timeout:      p.Timeout,
+			SetupTask:          p.Pre,
+			TeardownTask:       p.Post,
+			Timeout:            p.Timeout,
+			SetupGroupFailTask: p.Pre == nil || !p.ContinueOnPreError,
```

Comment:
I don't think the `p.Pre == nil` check is necessary.

Reply:
It is because p can be nil here